lib/engines/content_object_store/config/locales/en.yml

```diff
@@ -1,3 +1,5 @@
 en:
   activerecord:
     attributes:
+      content_object_store/content_block_edition/content_block_document:
```

Comment:
would you be able to ELI5 how this is used? Is there magic going on in the view somewhere?

Reply:
Rails magic! Which is why it was so hard to find the right combination.

[What I had to go on was this piece of the docs](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models), if you scroll down slightly you get to this:

```
In the event you need to access nested attributes within a given model, you should nest these under model/attribute at the model level of your translation file:

en:
  activerecord:
    attributes:
      user/role:
        admin: "Admin"
        contributor: "Contributor"
```

Which gave me the clue there was some magic `/` syntax to make it work for nested objects.

The trouble I had was how `content_object_store` came into the picture, i.e why not

```
content_object_store/content_block_edition/content_object_store/content_block_document:
or 
content_object_store/content_block_edition/content_block_document_attributes:
```